config/settings.yml

```diff
@@ -13,3 +13,5 @@ zotero_api:
   solr_document_field: zotero_bibliography_tsi
 sidekiq:
   logger_level: 'Logger::INFO' # The default from Sidekiq
+sync_bibliography_service:
+  enabled: true # FeatureFlag: SyncBibliograhyService
```

Comment:
Any thoughts on setting this to `false` by default and setting it to `true` in test/dev?

The only reason I mention that, is that this (confusingly) checked in file will get overrwritten by what's in [`shared_configs`](https://github.com/sul-dlss/shared_configs/blob/exhibits-prod/config/settings/production.yml) and could accidentally lead us to releasing the feature in production.